eZ/Publish/API/Repository/Tests/ContentServiceAuthorizationTest.php

```diff
@@ -1626,13 +1673,54 @@ public function testCopyContentToAuthorizedLocation()
         $policyCreateStruct->addLimitation($locationLimitation);
         $roleCreateStruct->addPolicy($policyCreateStruct);
 
-        // check if content/publish policy is available (@since 6.8)
-        $limitations = $roleService->getLimitationTypesByModuleFunction('content', 'publish');
-        if (array_key_exists('Node', $limitations)) {
-            $policyCreateStruct = $roleService->newPolicyCreateStruct('content', 'publish');
-            $policyCreateStruct->addLimitation($locationLimitation);
-            $roleCreateStruct->addPolicy($policyCreateStruct);
-        }
+        $roleDraft = $roleService->createRole($roleCreateStruct);
+        $roleService->publishRoleDraft($roleDraft);
+
+        // Create a user with that Role
+        $user = $this->createCustomUserVersion1('Users', $roleIdentifier);
+        $repository->getPermissionResolver()->setCurrentUserReference($user);
+
+        // Test copying Content to the authorized Location
+        $contentService->copyContent(
+            $authorizedFolder->contentInfo,
+            $locationService->newLocationCreateStruct(
+                $authorizedFolder->contentInfo->mainLocationId
+            )
+        );
+    }
+
+    /**
+     * Test copying Content to the authorized Location (limited by policies).
+     */
+    public function testCopyContentToAuthorizedLocationWithSubtreeLimitation()
+    {
+        $repository = $this->getRepository();
+        $contentService = $repository->getContentService();
+        $locationService = $repository->getLocationService();
+        $roleService = $repository->getRoleService();
+
+        // Create and publish folders for the test case
+        $folderDraft = $this->createContentDraft('folder', 2, ['name' => 'Folder1']);
+        $contentService->publishVersion($folderDraft->versionInfo);
+        $authorizedFolderDraft = $this->createContentDraft('folder', 2, ['name' => 'AuthorizedFolder']);
+        $authorizedFolder = $contentService->publishVersion($authorizedFolderDraft->versionInfo);
+
+        // Prepare Role for the test case
+        $roleIdentifier = 'authorized_subree';
+        $roleCreateStruct = $roleService->newRoleCreateStruct($roleIdentifier);
+        $subtreeLimitation = new SubtreeLimitation(
+            ['limitationValues' => ['/1/2']]
+        );
+        $policyContentRead = $roleService->newPolicyCreateStruct('content', 'read');
```

Comment:
There is a syntax sugar for this: `\eZ\Publish\API\Repository\Tests\BaseTest::createRoleWithPolicies`

Reply:
@alongosz sugar syntax were applied